blockly-vue-advanced/src/components/BlocklyComponent.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+  <div>
+
+    <div id="blocklyDiv" txt1="test"></div>
+  </div>
+</template>
+
+<script>
+/**
+ * @license
+ *
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Blockly Vue Component.
+ * @author abbychau@gmail.com (Abby Chau)
+ */
+
+import Blockly from "blockly";
+import Swal from "sweetalert2";
+
+var renameVar = function(name) {
+  return name;
+};
+
+Blockly.prompt = function(msg, defaultValue, callback) {
```

Comment:
Can we organize this a little. My suggestion would be to move blocks into a separate file. And possibly the prompt into a blockly_configure.js file or something like that?

Reply:
This would be a good idea. I am on a trip so I can move it a few days later.

Reply:
No problem, take your time, and thanks again!